superset-frontend/src/views/components/RightMenu.tsx

```diff
@@ -281,7 +281,7 @@ const RightMenu = ({
           onDatabaseAdd={handleDatabaseAdd}
         />
       )}
-      {environmentTag.text && (
+      {environmentTag && environmentTag.text && (
```

Comment:
Could we simplify this to
```suggestion
      {environmentTag?.text && (
```

Reply:
Oh, too late 🙁

Reply:
Ops...Let's put it on the next PR

Reply:
> Ops...Let's put it on the next PR

no worries, it was just a minor nit 🙂👍